dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTableValueCustom.java

```diff
@@ -42,7 +42,9 @@ public Ds3TreeTableValueCustom(final String bucketName, final String name, final
         this.searchOn = searchOn;
     }
 
-    public boolean isContainer() { return (type == Ds3TreeTableValue.Type.Bucket || type == Ds3TreeTableValue.Type.File); }
+    public boolean isContainer() {
```

Comment:
Can we get a comment here telling us what the concept of a container is?  Trying to figure out the difference between bucket & file both being a container.  Naively, I'd intuit that a bucket is a container, because it is folder like, where a file wouldn't be a container, because a file I'd intuit to be not folder like.